Assignment 1 (question 1)

Suruchibharti opened the pull request:
"Submitted Assignmnet-1 (Question-1) on 2nd October"

Suruchibharti committed:
Create Calculator.c

functions added for calculator

Suruchibharti committed:
Update Calculator.c

one more module added calc()

Suruchibharti committed:
Update Calculator.c

Comments added for each function for better understanding of the code

arti-niraj-yadav12 reviewed (changes requested):
As per Requirements The program should handle whitespace between numbers and operators, which should be ignored.
and
The program should follow the order of operations (DMAS).
but your code is not following above requirements.

Suruchibharti committed:
Update Calculator.c

Resolved bugs and updated Calculator.c

Suruchibharti commented:
> As per Requirements The program should handle whitespace between numbers and operators, which should be ignored. and The program should follow the order of operations (DMAS). but your code is not following above requirements.

I’ve updated the program to ignore whitespaces between numbers and operators and implemented proper operator precedence according to DMAS in Calculator.c file

arti-niraj-yadav12 reviewed (changes requested):
Give meaningful names for variables

Suruchibharti committed:
Update Calculator.c

I have renamed all the variables to meaningful names

arti-niraj-yadav12 commented on Calculator.c:
```diff
@@ -0,0 +1,123 @@
+
+//use to check operator
+int isOperator(char c) {
+    return (c=='+' || c=='-' || c=='*' || c=='/');
```
what is c?
Give proper naming convention to  variables

arti-niraj-yadav12 commented on Calculator.c:
```diff
@@ -0,0 +1,123 @@
+}
+
+//use for evaluate the operation b/w two operand
+int performOperation(int x, int y, char c, int *err) {
```
what is x,y,c, err?
Give meaningful names to variables

Suruchibharti committed:
Update Calculator.c

Renamed variables with meaningful names and use camelcase for better code readability
